packages/config/config/devices/0x0085/zws230.json

```diff
@@ -0,0 +1,29 @@
+// Fakro ZWS230
+// Chain actuator - window opener
+{
+	"manufacturer": "Fakro",
+	"manufacturerId": "0x0085",
+	"label": "ZWS230",
+	"description": "Chain actuator 230VAC",
+	"devices": [
+		{
+			"productType": "0x0002",
+			"productId": "0x0003"
+		},
+		{
+			"productType": "0x0003",
+			"productId": "0x0002"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 5,
+			"isLifeline": true
+		}
+	}
```

Comment:
Do you know if the device supports Z-Wave Plus? In that case this definition shouldn't be necessary.

Reply:
I think these are not Zwave Plus devices, there is no mention of "Plus" in their manuals.

Also, although it has a switch_multilevel, you cannot really open it partially that way, any value above 0 opens the window all the way. The device has to be refreshed to see the new opening level (which immediatelly jumps to max btw),

Reply:
From your dump:
```
  "zwavePlusVersion": 1,
```
Guess it doesn't hurt to leave this in though.